py3status/modules/deadbeef.py

```diff
@@ -3,95 +3,101 @@
 Display song currently playing in deadbeef.
 
 Configuration parameters:
-    cache_timeout: how often we refresh usage in seconds (default 1)
-    delimiter: delimiter character for parsing (default '¥')
-    format: see placeholders below (default '{artist} - {title}')
+    cache_timeout: refresh interval for this module (default 1)
+    delimiter: delimiter between metadata fields (default '¥å¥å¥å')
+    format: display format for this module (default '{artist} - {title}')
 
 Format placeholders:
-    {artist} artist
-    {title} title
-    {elapsed} elapsed time
-    {length} total length
-    {year} year
-    {tracknum} track number
+    {artist}        artist
+    {length}        length
+    {playback_time} playback time
+    {title}         title
+    {tracknumber}   track number
+    {year}          year
 
 Color options:
-    color_bad: An error occurred
+    color_degraded: Paused
+    color_good: Playing
+    color_bad: Not Playing
 
 Requires:
-        deadbeef:
+    deadbeef: a GTK+ audio player for GNU/Linux
 
 @author mrt-prodz
 """
-from subprocess import check_output, CalledProcessError
+
+import subprocess
 
 
 class Py3status:
     # available configuration parameters
     cache_timeout = 1
-    delimiter = u'¥'
+    delimiter = u'¥å¥å¥å'
```

Comment:
What I need from `self.py3.command_output` is to ignore error codes regardless since I'm using `pgrep deadbeef` to see if it's running. Like I said, starting this can generate lot of startup noises and error codes.... Even running `deadbeef --nowplaying-tf %artist%^^%title%` can slightly change how deadbeef GUI looks onscreen (sometimes it opens an "empty" tab).. or we use subprocess as-is. Will you look into doing same thing for `mpd_status` with this? Full >100 fields. I will test, no problem.